### PR TITLE
[OSDOCS 6508] Restructure install config parms to book level IBM Z 

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-6508
https://issues.redhat.com/browse/OSDOCS-6509

Link to docs preview:

- [Installation configuration parameters for IBM Z](https://62676--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installation-config-parameters-ibm-z.html)
New assembly at book level
- Installing a cluster on IBM Z [Manually creating the installation configuration file](https://62676--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installation-initializing-manual_installing-ibm-z)
Note the absence of Installation configuration parameters following this topic. In liue of the module, a cross reference is provided for the new assembly.
It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all IBM Z  installation assemblies.

QE review:
- [NA ] QE has approved this change.
  Given that this update is a reorg only, QE approval is not required. See conversation here: https://github.com/openshift/openshift-docs/pull/61165#issuecomment-1600861041. 

Additional information: 
- Related PR https://github.com/openshift/openshift-docs/pull/61165

### CODE REVIEW

**Comment by SNiemann15** [comment]
/label peer-review-needed

@sjstout PTAL

**Comment by stevsmit** [comment]
I don't have a lot of feedback for this, as the content looks to be migrated correctly. I went through each of the assemblies and modules to make sure things were there, and it looks good to me. I did leave one note because of an extra space, but I think the content is to go for merge review.

File: installing/installing_ibm_z/installation-config-parameters-ibm-z-kvm.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="installation-config-parameters-ibm-z-kvm"]
+= Installation configuration parameters for IBM Z(R) and {linuxoneProductName} with {op-system-base} KVM
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply installing/installing_ibm_z/installation-config-parameters-ibm-z-kvm.adoc:3]: Yes first use of IBM Z should contain a registered trademark symbol see recent changes I made to the SSG [here](https://redhat-documentation.github.io/supplementary-style-guide/#ibm-z).

File: installing/installing_ibm_z/installation-config-parameters-ibm-z-kvm.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+include::_attributes/common-attributes.adoc[]
+:context: installation-config-parameters-ibm-z-kvm
+:platform: IBM Z
+
+toc::[]
+
+Before you deploy an {product-title} cluster on {ibmzProductName}, you provide parameters to customize your cluster and the platform that hosts it. When you create the `install-config.yaml` file, you provide values for the required parameters through the command line. You can then modify the `install-config.yaml` file to customize your cluster further.
+
+[NOTE]
```
Comment (mjpytlak) [inline installing/installing_ibm_z/installation-config-parameters-ibm-z-kvm.adoc:12]: @SNiemann15 Is this a holdover from when you were going to use a single assembly twice in the TOC. Given that we now have two separate assemblies, I am not sure if the note is required. You could always add it back when you deliver https://issues.redhat.com/browse/OSDOCS-7108

File: modules/installation-configuration-parameters.adoc [hunk 3]
```diff
@@ -58,6 +54,7 @@
 // * installing/installing-restricted-networks-nutanix-installer-provisioned.adoc
 // * installing/installing_vsphere/installation-config-parameters-vsphere.adoc
 // * installing/installing_gcp/installation-config-parameters-gcp.adoc
+// * installing/installing_gcp/installation-config-parameters-ibm-z.adoc
```
Comment (mjpytlak) [inline modules/installation-configuration-parameters.adoc:57]: @SNiemann15 Please update the Where used list to reflect that this module now appears in the second assembly you created.

**Pull request merged**